.devcontainer/devcontainer.json

```
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/miniconda
+{
+    "name": "Miniconda (Python 3)",
+    "image": "continuumio/miniconda3",
+
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    // "features": {},
+
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    // "forwardPorts": [],
+
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": "bash /workspaces/conda/.devcontainer/post_create.sh",
+    "postStartCommand": "/opt/conda/bin/python -m conda init --dev bash",
+
+    // Configure tool-specific properties.
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "charliermarsh.ruff",
+                "eamodio.gitlens",
+                "ms-toolsai.jupyter",
+                "lextudio.restructuredtext-pack",
+                "lextudio.restructuredtext@prerelease",
+                "swyddfa.esbonio@prerelease",
+                "ms-azuretools.vscode-docker"
+            ]
+        }
+    }
+
+    // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

We are already connecting as root anyway, right?

---

Yes. This only becomes relevant when the default user is a different one, for example because the devcontainer is based on another container that uses a different user or when making use of the [common-utils feature](https://github.com/devcontainers/features/tree/main/src/common-utils).

---

In that case I'd change the comment, because right now that "instead" is a bit misleading.
```suggestion
    // Adjust to connect as non-root instead. More info: https://aka.ms/dev-containers-non-root.
```